Cargo.toml

```diff
@@ -274,7 +274,7 @@ anvil_zksync_api_server = { git = "https://github.com/matter-labs/anvil-zksync.g
 anvil_zksync_common = { git = "https://github.com/matter-labs/anvil-zksync.git", rev = "ff18d025d7f278a2389b6f9f0266917570d727a3" }
 anvil_zksync_traces = { git = "https://github.com/matter-labs/anvil-zksync.git", rev = "ff18d025d7f278a2389b6f9f0266917570d727a3" }
 anvil_zksync_l1_sidecar = { git = "https://github.com/matter-labs/anvil-zksync.git", rev = "ff18d025d7f278a2389b6f9f0266917570d727a3" }
-zksync_telemetry = { git = "https://github.com/matter-labs/zksync-telemetry.git", rev = "d5c35951843263765079545c31c908dcbaab1f30" }
+zksync_telemetry = { git = "https://github.com/matter-labs/zksync-telemetry.git", rev = "f6d8618d870a09467ff24ea32ef57e01af8f311e" }
```

Comment:
Note that this has currently no effect as we still have the patch which requires https://github.com/matter-labs/anvil-zksync/pull/657 to be merged to be removed